resources/scripts/copy_node_logs.sh

```diff
@@ -10,4 +10,4 @@ else
 fi
 
 # Find all .log files and copy them to tmpdir preserving the same directory structure
-rsync -avm --include='*.log*' -f 'hide,! */' ~/.local/share/safe/node/ tmpdir/
\ No newline at end of file
+rsync -avm --include='*.log*' -f 'hide,! */' ~/var/log/safenode/ tmpdir/
```

Comment:
I think it should not be under HOME?
```suggestion
rsync -avm --include='*.log*' -f 'hide,! */' /var/log/safenode/ tmpdir/
```